.github/workflows/chrome_v112.yml

```
@@ -0,0 +1,20 @@
+name: chrome_v112
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+      - uses: browser-actions/setup-chromium@v1
+        with:
+          chrome-version: "112001"
```

Use branch position for v112 from https://chromiumdash.appspot.com/releases?platform=Linux
```suggestion
          chrome-version: "1109224"
```